packages/pangea-sdk/pangea/asyncio/services/audit.py

```diff
@@ -361,6 +362,7 @@ async def results(
             id (string): the id of a search action, found in `response.result.id`
             limit (integer, optional): the maximum number of results to return, default is 20
             offset (integer, optional): the position of the first result to return, default is 0
+            assert_search_restriction (dict, optional): A list of keys to check against the original search request. Ensuring the requested search results followed the same expected search restrictions.
```

Comment:
```suggestion
            assert_search_restriction (dict, optional): For each key/value pair in the dictionary, ensure the key exists in each log record and the value in the log record matches the value list in the search restriction dictionary
```

Reply:
Updated the text too: `Assert the requested search results were queried with the exact same search restrictions, to ensure the results comply to the expected restrictions.`. Which I think is more accurate as this feature is just checking that the assert_search_restrictions exactly matches the search_restrictions passed in the original search

Reply:
I like it